### PR TITLE
Use path parameters in workstation session creation URLs

### PR DESCRIPTION
**Pull request by HarmvZ**
Part of https://github.com/DIAGNijmegen/rse-roadmap/issues/225.

This PR changes the workstation session creation URLs to use path parameters instead of query parameters. 

**This is ready for review, but it cannot be released until the changes at the cirrus side are finalized and released.**

### CODE REVIEW

File: app/grandchallenge/workstations/urls.py
Comment (jmsmkn) [inline app/grandchallenge/workstations/urls.py:35]: Each view needs its own unique name

**Comment by jmsmkn** [comment]
You will need to merge in main to fix the codecov problem.

File: app/grandchallenge/workstations/static/workstations/js/session-control.mjs
Comment (jmsmkn) [inline app/grandchallenge/workstations/static/workstations/js/session-control.mjs:62]: This means that we cannot merge this PR until CIRRUS 2023.05 is released, and CIRRUS 2023.04 is set as the previous version, right?

File: app/grandchallenge/workstations/templatetags/workstations.py
Comment (jmsmkn) [inline app/grandchallenge/workstations/templatetags/workstations.py:68]: A NamedTuple should be used when returning multiple values from a function.

File: app/grandchallenge/workstations/urls.py
Comment (jmsmkn) [reply app/grandchallenge/workstations/urls.py:35]: Oh, I see, for the URL dispatcher.

File: app/grandchallenge/workstations/urls.py
Comment (jmsmkn) [inline app/grandchallenge/workstations/urls.py:33]: This should be able to handle any changes to the CIRRUS URLS:

```suggestion
        "<slug>/sessions/create/<path:workstation_path>",
```

File: app/grandchallenge/workstations/urls.py
Comment (jmsmkn) [inline app/grandchallenge/workstations/urls.py:45]:
```suggestion
        "<slug>/sessions/debug/create/<path:workstation_path>",
        DebugSessionCreate.as_view(),
        name="workstation-debug-session-create",
```

File: app/grandchallenge/workstations/static/workstations/js/session-control.mjs
Comment (HarmvZ) [reply app/grandchallenge/workstations/static/workstations/js/session-control.mjs:62]: Correct. The changes from https://github.com/DIAGNijmegen/rse-cirrus-core/pull/2415 need to be released in the new CIRRUS version before this PR can be merged and released.

**Review by jmsmkn — APPROVED** [review]

**Pull request merged**
